lineage/figures/figure6B.py

```diff
@@ -0,0 +1,65 @@
+""" This file contains functions for plotting the performance of the model for censored data. """
+
+import numpy as np
+
+from ..Analyze import run_Analyze_AIC
+from ..LineageTree import LineageTree
+from ..data.Lineage_collections import Gemcitabine_Control
+
+# States to evaluate with the model
+from ..states.StateDistributionGaPhs import StateDistribution
+from .figureCommon import getSetup, lineage_good_to_analyze, subplotLabel
+from .figureS10 import run_AIC, figure_maker
+
+
+desired_num_states = np.arange(1, 8)
+
+
+def makeFigure():
+    """
+    Makes figure 10.
+    """
+    ax, f = getSetup((13.333, 3.333), (1, 4))
+    desired_num_states = np.arange(1, 8)
+
+    # Setting up state distributions and E
+    Sone = StateDistribution(0.99, 0.9, 10, 2, 10, 2)
+    Stwo = StateDistribution(0.9, 0.9, 20, 3, 20, 3)
+    Sthree = StateDistribution(0.85, 0.9, 30, 4, 30, 4)
+    Sfour = StateDistribution(0.8, 0.9, 40, 5, 40, 5)
+    Eone = [Sone, Sone]
+    Etwo = [Sone, Stwo]
+    Ethree = [Sone, Stwo, Sthree]
+    Efour = [Sone, Stwo, Sthree, Sfour]
+    E = [Eone, Etwo, Ethree, Efour]
```

Comment:
I don't think these lines of code are needed. Should probably be removed.

Reply:
Sorry line 23 is needed. I don't think the rest are.

Reply:
Never mind, I just noticed you wanted this on master before investigating further.